custom_components/tuya_local/devices/petempo_paf02_pet_feeder.yaml

```diff
@@ -0,0 +1,96 @@
+name: Petempo PAF-02 pet feeder
```

Comment:
Top level name should be generic in casse other brands also match the config. Branding should be kept under products.